_data/tools.json

```diff
@@ -989,6 +989,15 @@
       "note": null,
       "type": "SAST"
    },
+   {
+      "title": "Fluid Attack's Scanner",
+      "url": "https://docs.fluidattacks.com/machine/scanner",
+      "owner": "Fluid Attacks",
+      "license": "Open Source",
+      "platforms": null,
+      "note": "SAST, DAST and SCA vulnerability detection tool with perfect OWASP Benchmark score.",
```

Comment:
@davewichers are you aware of this? Is the claim supportable?

Reply:
- Code is open source and you can reproduce results as explained here: https://docs.fluidattacks.com/machine/scanner/reproducibility
- @davewichers is aware: https://github.com/OWASP/Benchmark/pull/146#issuecomment-825996304
  but has no time for the project I guess

Reply:
@kingthorin Is this enough proof or should I remove the claim?

We have nothing to hide anyway, code is there: https://gitlab.com/fluidattacks/product/-/tree/master/skims

Also instructions to reproduce results, etc, etc

Reply:
Thanks!

Reply:
I’m happy with that background info 👍

Reply:
Thanks to you for the peer-review 👍🏻